cnc_ctrl_v1/Settings.cpp

```diff
@@ -77,7 +77,7 @@ void settingsReset() {
     sysSettings.originalChainLength = 1651;   // int originalChainLength;
     sysSettings.encoderSteps = 8113.73; // float encoderSteps;
     sysSettings.distPerRot = 63.5;   // float distPerRot;
-    sysSettings.maxFeed = 700;   // int maxFeed;
+    sysSettings.maxFeed = 900;   // int maxFeed;
```

Comment:
I'm not sure that we should be increasing this. Is this safe to do globally?

Reply:
if using webcontrol, it gets set from the .json file.